docs/index.md

```diff
@@ -1087,16 +1087,30 @@ Files specified using `--file` _are not affected_ by this option.
 
 Can be specified multiple times.
 
-### `--extension <ext>, --watch-extensions <ext>`
-
-> _Updated in v6.0.0. Previously `--watch-extensions`, but now expanded to affect general test file loading behavior. `--watch-extensions` is now an alias_
+### `--extension <ext>`
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
 Specifying `--extension` will _remove_ `.js` as a test file extension; use `--extension js` to re-add it. For example, to load `.mjs` and `.js` test files, you must supply `--extension mjs --extension js`.
 
+### `--watch-files <file|directory|glob>`
+
+> _New in v7.0.0
```

Comment:
The final "_" is missing: `_New in v7.0.0_`

Reply:
Fixed in https://github.com/mochajs/mocha/pull/3980/commits/10f4fda066ca96a242a3a69d52207e70664f1a01